src/eth/execute.yaml

```
@@ -54,3 +54,24 @@
         gasUsed:
           title: Gas used
           $ref: '#/components/schemas/uint'
+- name: eth_multicallV1
+  summary: Executes a sequence of message calls building on each other's state without creating transactions on the block chain, optionally overriding block and state data
+  params:
+    - name: Block state calls
+      schema:
+        title: Arguments for multi call
+        type: array
+        items:
+          $ref: '#/components/schemas/BlockStateCalls'
+    - name: Block tag
+      required: false
+      schema:
+        type: boolean
+    - name: Trace Transfers - Add ETH transfers as fake events to the event log response
+      required: false
+      schema:
+        $ref: '#/components/schemas/BlockNumberOrTagOrHash'
+  result:
+    name: Result of calls
+    schema:
+      $ref: '#/components/schemas/MultiCallResult'
```

```suggestion
- name: eth_multicallV1
  summary: Executes a sequence of message calls building on each other's state without creating transactions on the block chain, optionally overriding block and state data
  params:
    - name: Details
      required: true
      schema:
        - name: Calls
          schema:
            title: Arguments for multi call
            type: array
            items:
              $ref: '#/components/schemas/BlockStateCalls'
        - name: TraceTransfers
          required: false
          schema:
            $ref: boolean
        - name: Validation
          required: false
          schema:
            $ref: boolean
    - name: Block tag
      required: false
      schema:
        type: '#/components/schemas/BlockNumberOrTagOrHash'
  result:
    name: Result of calls
    schema:
      $ref: '#/components/schemas/MultiCallResult'
```

not sure if I understood the syntax correctly. But this is about wrapping the arguments into an object

---

There's also add to teh validation mode. We also need to agree on names for these

---

I believe an object needs `title`, `type: 'object'`, and `properties` on it, and the `properties` should then contain the list of properties.  I believe `schema` is used instead of these if the schema is defined elsewhere, but in this case you are defining it inline so there would be no `schema` property.